controls/treeview/commands.md

```diff
@@ -8,41 +8,44 @@ slug: treeview-commands
 
 # Commands
 
-The .NET MAUI TreeView provides various commands that allow you to initiate expand or collapse actions, check and uncheck actions, and to scroll to a specific item.
+The .NET MAUI TreeView provides various commands that allow you to initiate expand or collapse actions, check and uncheck actions, scroll to a specific item and more. The table below lists all available commands:
 
-## Expanding and Collapsing Items
+| Commands | Description |
+| -------- | ---------- |
+| `ExpandAllCommand` (`ICommand`) | Gets a command to expand all items in the control. |
+| `CollapseAllCommand` (`ICommand`) | Gets a command to collapse all items in the control. |
+| `CheckAllCommand` (`ICommand`) | Gets a command to check all items in the control. |
+| `UncheckAllCommand` (`ICommand`) | Gets a command to uncheck all items in the control. |
+| `ScrollToCommand` (`ICommand`) | Gets a command to uncheck all items in the control. |
+| `ItemExpandedCommand` (`ICommand`) | Defines a command to execute when an item is expanded after a user interaction. The command should accept a single parameter with the expanded item. |
+| `ItemCollapsedCommand` (`ICommand`) | Defines a command to execute when an item is collapsed after a user interaction. The command should accept a single parameter with the collapsed item. |
+| `ItemCheckedCommand` (`ICommand`) | Defines a command to execute when an item is checked after a user interaction. The command should accept a single parameter with the checked item. |
+| `ItemUncheckedCommand` (`ICommand`) | Defines a command to execute when an item is unchecked after a user interaction. The command should accept a single parameter with the unchecked item. |
+| `LoadChildrenOnDemandCommand` (`ICommand`) | Defines a command to execute when loading an item on demand. The command accepts a single parameter of type `Telerik.Maui.Controls.TreeView.TreeViewLoadChildrenOnDemandCommandContext`. |
+| `ItemTappedCommand` (`ICommand`) | Defines a command to execute when an item is tapped. The command accepts a single parameter with the item being tapped. |
+| `ItemHoldingCommand` (`ICommand`) | Defines a command to execute when an item is held. The command accepts a single parameter with the item being held. |
 
-TreeView exposes the following commands for expanding and collapsing all items:
 
-* `ExpandAllCommand`(`ICommand`)&mdash;Gets a command to expand all items in the control.
-* `CollapseAllCommand`(`ICommand`)&mdash;Gets a command to collapse all items in the control.
+### Example with ExpandAll and CollapseAll Commands
```

Comment:
- Don't skip heading levels.
- Use complete sentences to introduce the snippets.